src/commands/dev/lib/dev-server.ts

```diff
@@ -435,7 +435,7 @@ export default class DevServer {
         this.output.debug(
           `Rebuilding asset "${entrypoint}" for "${req.method} ${req.url}"`
         );
-        buildPromise = executeBuild(nowJson, this, asset);
+        buildPromise = executeBuild(nowJson, this, asset, req.url);
```

Comment:
Would you rather have the query params from the URL included or not included?

Reply:
I'd lean towards no, because in Next the query shouldn't affect the build output. Maybe we should pass it in another property or just the whole `req`/`res` objects?

Reply:
We rely on some query parameters for rendering, so included would be great!